### PR TITLE
Add client unit test in L0_http / L0_grpc

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

**Review by CoderHam — DISMISSED** [review]

**Comment by tanmayv25** [comment]
What are these tests for? What are the clients being used in the test?

**Comment by GuanLuo** [comment]
@tanmayv25 This is for https://github.com/triton-inference-server/client/pull/71, currently it is just unit testing the multi infer API, but I gave it a generic name as ideally we should adopt more on unit testing the C++ client API instead of creating additional sample clients, in this way it will be more modularized and easier to extend test coverage.

**Review by CoderHam — DISMISSED** [review]

**Review by tanmayv25 — DISMISSED** [review]

**Review by CoderHam — APPROVED** [review]

**Review by tanmayv25 — APPROVED** [review]

File: qa/L0_grpc/test.sh [hunk 1]
```diff
@@ -82,8 +83,11 @@ if [[ "$(< /proc/sys/kernel/osrelease)" == *microsoft* ]]; then
     SIMPLE_IMAGE_CLIENT=${SDKDIR}/python/image_client
     # SIMPLE_ENSEMBLE_IMAGE_CLIENT=${SDKDIR}/python/ensemble_image_client
     SIMPLE_REUSE_INFER_OBJECTS_CLIENT=${SDKDIR}/python/reuse_infer_objects_client
+    # [FIXME] point to proper client
```
Comment (rmccorm4) [inline qa/L0_grpc/test.sh:86]: Do these FIXME's need changes?

File: qa/L0_http/test.sh [hunk 2]
```diff
@@ -73,8 +74,11 @@ if [[ "$(< /proc/sys/kernel/osrelease)" == *microsoft* ]]; then
     SIMPLE_SHM_CLIENT=${SDKDIR}/python/simple_http_shm_client
     SIMPLE_CUDASHM_CLIENT=${SDKDIR}/python/simple_http_cudashm_client
     SIMPLE_REUSE_INFER_OBJECTS_CLIENT=${SDKDIR}/python/reuse_infer_objects_client
+    # [FIXME] point to proper client
```
Comment (rmccorm4) [inline qa/L0_http/test.sh:77]: Do these FIXME's need changes?

File: qa/L0_grpc/test.sh (hunk 1 above)
Comment (GuanLuo) [reply qa/L0_grpc/test.sh:86]: Not in this change, this should be taken care of when we port this test to Windows platform as well, note that the paths for the other clients above also need to be fixed ("python" path for cpp clients)

**Pull request merged**